composer.json

```diff
@@ -20,7 +20,7 @@
         }
     ],
     "require": {
-        "php": ">=5.3.3",
+        "php": ">=5.5",
```

Comment:
I mean, I don't think it makes sense to bump all adapters to 0.5 but not the main lib (as if we tag this as 0.5, then tag slmqueue as 0.5, the adapters would need to be tagged 0.6).

Envoyé de mon iPhone

> Le 12 août 2015 à 17:37, Bas Kamer <notifications@github.com> a écrit :
> 
> In composer.json:
> 
> > @@ -20,7 +20,7 @@
> >          }
> >      ],
> >      "require": {
> > -        "php": ">=5.3.3",
> > +        "php": ">=5.5",
> Are u saying I should not bumping requirement in this PR until 0.5 is done? Or just pointing out 0.5 work is still ahead?
> 
> —
> Reply to this email directly or view it on GitHub.
>